demo/src/app/app.routes.server.ts

```diff
@@ -0,0 +1,9 @@
+import { RenderMode, ServerRoute } from '@angular/ssr'
+
+export const serverRoutes: ServerRoute[] = [
+  { path: 'detail/:id', renderMode: RenderMode.Server },
```

Comment:
Without providing this configuration Angular builds alone were failing with:

> ✘ [ERROR] The 'detail/:id' route uses prerendering and includes parameters, but 'getPrerenderParams' is missing. Please define 'getPrerenderParams' function for this route in your server routing configuration or specify a different 'renderMode'.

Note - this would be good opportunity to test Prerendering with fallback ( https://ng-dev-previews-fw--pr-angular-angular-58445-adev-prev-gwy7p64b.web.app/guide/hybrid-rendering#prerender-fallback-strategies), but it currently doesn't actually work:

```
⠴ Building...
✘ [ERROR] No matching export in "node_modules/@angular/ssr/fesm2022/ssr.mjs" for import "PrerenderFallback"

    src/app/app.routes.server.ts:2:21:
      2 │ import { RenderMode, PrerenderFallback } from '@angular/ssr';
        ╵                      ~~~~~~~~~~~~~~~~~

⠦ Building...
```

(TS types also suggest this named export should exist)